doc/sources/oneapi_gpu.rst

```diff
@@ -0,0 +1,100 @@
+.. ******************************************************************************
+.. * Copyright 2020-2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _oneapi_gpu:
+
+##############################################################
+oneAPI and GPU support in Intel(R) Extension for Scikit-learn*
+##############################################################
+
+Intel(R) Extension for Scikit-learn* provides support of oneAPI concepts, which
+means that algorithms can be executed on different devices - CPUs and GPUs.
+This is done by integration with
+`dpctl <https://intelpython.github.io/dpctl/latest/index.html>`_ package, that
+implements core of oneAPI concepts like queues and devices.
+
+Intel(R) Extension for Scikit-learn* offers two ways how to run an algorithm on
+specific device with help of dpctl:
+
+
+1. Pass input data as
+   `dpctl.tensor.usm_ndarray
+   <https://intelpython.github.io/dpctl/latest/docfiles/dpctl.tensor_api.html#dpctl.tensor.usm_ndarray>`_
+   to the algorithm. The computation will run on the device where input data is
+   located, and result will be returned as :code:`usm_ndarray` on the same
+   device.
+
+   .. note::
+     All the data passed to the algorithm must reside on the same device.
+
+   .. warning::
+     Only base methods like :code:`fit`, :code:`predict`, and :code:`transform`
+     can consume :code:`usm_ndarray`. Please pay attension that only the
+     algorithms in Intel(R) Extension for Scikit-learn* provide support of
+     :code:`usm_ndarray`. Standard algorithms in stock version of Scikit-learn*
+     do not include this feature.
+2. Use global configurations of Intel(R) Extension for Scikit-learn*:
+     1. Option :code:`target_offload` can be used to set the device primarily
+        used to perform computations. Accepted values are :code:`str` and
+        :code:`dpctl.SyclQueue`. If string, expected to be "auto" (the execution
+        context is deduced from input data location), or SYCL* filter selector
+        string. Default value is "auto"
+     2. Option :code:`allow_fallback_to_host`
+        is a boolean flag, that, if set, allows to fallback computation to host
+        device in case when particular estimator does not support the selected
+        one. Default value is :code:`False`.
```

Comment:
`Accepted values` -- you really mean `accepted data types`, no?

I also don't think my changes below are correct, because it's not clear what happens if the input is dpctl.SyclQueue.

```suggestion
     1. The :code:`target_offload` option can be used to set the device primarily
        used to perform computations. Accepted values are :code:`str` and
        :code:`dpctl.SyclQueue`. The default value is "auto", meaning that the execution
        context is deduced from the location of input data. Otherwise, a string with a
        SYCL* filter selector is expected.
     2. The :code:`allow_fallback_to_host` option
        is a Boolean flag. If set to :code:`True`, the computation is allowed to fallback to the host
        device when a particular estimator does not support the selected
        device. The default value is :code:`False`.
```

Reply:
`I also don't think my changes below are correct, because it's not clear what happens if the input is dpctl.SyclQueue.`
It means that if the data type is a string, then it can take the values "auto" or SYCL * filter selector. For the dpctl type.SyclQueue there are no restrictions